Adds support for remote serial ports.

legacycode opened the pull request:
Hi, it would be great to have remote serial ports also included in this library. With this PR i added a factory that checks the prefix of the port property. If the port starts with "rfc2217://" it will use the RFC2217 implementation of serial else the serial port implementation. I have testet this with ser2net and the following config line:

`2000:telnet:0:/dev/ttyS0:9600 remctl banner`

You can then run the following command from the client command line:

`pylacrosse -d rfc2217://192.168.0.1:20001 scan`

I am not a python developer. Feel free to change this code :-)

legacycode commented:
Maybe this is also an alternative to #5

hthiery commented:
looks good so far ... could you please add a note in the README to make this new feature visible

legacycode commented:
I removed the unused import and added documentation. Hopefully it is clear to understand. It would be great, if you update your homeassistant component as well :-)

hthiery commented:
Just published a new version (0.4) of the lib. Don't know when I have the time to update it in homeassistant. Maybe you're interested in doing that? Please feel free.

legacycode commented:
Done with PR: https://github.com/home-assistant/home-assistant/pull/26088
